config/install/field.field.user.user.message_notify_number.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
```

Comment:
We'd prefer that telephone number fields not get installed automatically. Especially as you may be introducing fields with unwanted machine names, modifying entity displays, or the field may already exist.
Despite all the field storage and displays being created, the user would still have to create a phone_number_settings config entity.
The phone number system for SMS framework actually has a UI wizard to create telephone fields, or you can match existing fields with whatever IDs. Some third party modules use their own field type even.
So a vote here to remove this `message_notify_number` field, and not modify entity form and entity view displays (`\message_notify_install()`).
Instructions should be provided to site builders on how to configure SMS Framework to work with this plugin. These steps are standard SMS Framework configuration steps, not unique to message_notify.

Reply:
Yep, was wondering about this while reading [the docs](https://www.drupal.org/docs/8/modules/sms-framework/integrating-with-sms-framework/sending-sms-with-apis). The page mentions "sending SMS to entities", but I did not find anything about the entity gains a phone number.

Might be a good addition to that page, explaining how numbers get to entities.